backside/serverless.yml

```diff
@@ -0,0 +1,27 @@
+service: backside
+
+frameworkVersion: '2'
+
+provider:
+  name: aws
+
+plugins:
+  - serverless-offline
+
+package:
+  exclude:
+    - ./**
+  include:
+    - ./bin/**
+
+functions:
+  hello:
+    runtime: go1.x
+    handler: bin/hello
+    events:
+      - http: GET hello
+  world:
+    runtime: go1.x
+    handler: bin/world
+    events:
+      - http: GET world
```

Comment:
After running this I got the warning:
```
Serverless: Configuration warning:
Serverless:   at 'functions.hello.events[0].http': value 'GET hello' does not satisfy pattern /^(?:\*|(GET|POST|PUT|PATCH|OPTIONS|HEAD|DELETE|ANY) (\/\S*))$/i
Serverless:   at 'functions.world.events[0].http': value 'GET world' does not satisfy pattern /^(?:\*|(GET|POST|PUT|PATCH|OPTIONS|HEAD|DELETE|ANY) (\/\S*))$/i
```

I think it should be `GET /world` instead. The same applies for the `GET /hello` (notice the slash difference).